### PR TITLE
Make PEP8 compliant

### PR DESCRIPTION
**Pull request by habi**
As part of http://24pullrequests.com/ and to test the waters with the
Nikolai team

### CODE REVIEW

File: dodo.py [hunk 1]
```diff
@@ -11,7 +11,8 @@
 
 
 def recursive_glob(path, pattern):
-    """recursively walk path directories and return files matching the pattern"""
+    """recursively walk path directories and return files matching the
```
Comment (Kwpolska) [inline dodo.py]: That does not match PEP257 though, which is another important standard.

File: dodo.py [hunk 2]
```diff
@@ -41,7 +43,8 @@ def set_nikola_test_locales():
             locales = []
             languages = set()
             for line in out.splitlines():
-                if (line.endswith('.utf8') or line.endswith('.UTF-8')) and '_' in line:
+                if (line.endswith('.utf8') or line.endswith('.UTF-8')) and \
```
Comment (Kwpolska) [inline dodo.py]: We prefer to use (parentheses) for continuations.

File: setup.py [hunk 3]
```diff
@@ -90,7 +90,8 @@ def expands_symlinks_for_windows():
     sys.path = oldpath
     del sys.modules['winutils']
     if failures != -1:
-        print('WARNING: your working copy is now dirty by changes in samplesite, sphinx and themes')
+        print('WARNING: your working copy is now dirty by changes in'
+              'samplesite, sphinx and themes')
```
Comment (Kwpolska) [inline setup.py]: Missing a space between the two lines..

**Comment by Kwpolska** [comment]
Also, we don’t honor the 79 character limit.

**Comment by habi** [comment]
I've fixed the missing space and reverted changes in dodo.py.
Hope that still helps.

**Comment by Kwpolska** [comment]
Yeah, thanks for your contribution! :clap:

**Comment by habi** [comment]
Very welcome.
You mention that you don't honour the 79 characters limit.
I'll try to push some changes for `pep8 */*.py --ignore=E501` if that's OK.

**Comment by Kwpolska** [comment]
Actually, we [already do that](https://github.com/getnikola/nikola/blob/master/setup.cfg#L5).

**Comment by habi** [comment]
Didn't see that yet. There are bunch of PEP8 errors in the `tests` folder. I'll fix them and make another PR.

**Pull request merged**